kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food/store/GoodGrainsProductRepository.java

```diff
@@ -0,0 +1,51 @@
+package com.kodilla.good.patterns.food.store;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class GoodGrainsProductRepository {
+
+    public Map<Product, Integer> products = new HashMap<Product, Integer>();
+
+    public GoodGrainsProductRepository() {
+        products.put(new Product("Whole wheat bread", 5.60), 150);
+        products.put(new Product("Graham bread", 6.20 ), 80);
+        products.put(new Product("Tatarczuch bread", 11.50), 20);
+    }
+
+    public Map<Product, Integer> getProducts() {
```

Comment:
gdzie uzywasz tej metody? i po co? tu jakbys wyciagnal cala tabelke z bazy danych (cos jak select * from products), a to w tym zadaniu kompletnie niepotrzebne.